bin/xud

```diff
@@ -53,6 +53,18 @@ const { argv } = require('yargs')
       describe: 'Data directory for lnd',
       type: 'string',
     },
+    'lnd.certificate': {
+      describe: 'Certificate for lnd',
```

Comment:
Maybe call this `certpath` so it's more clear what this is used for just from the name. Also I think the description could be more clear as "Path to the SSL certificate for lnd."